ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/BindingSignature.cs

```diff
@@ -8,8 +9,18 @@ internal partial class BindingSignature
 
     public void SetIsMutable(bool value)
     {
-      if (!value)
-        throw new System.NotImplementedException();
+      if (!value && MutableKeyword != null)
+      {
+        if (MutableKeyword.PrevSibling is Whitespace whitespace)
```

Comment:
We can also try not disabling the formatter for this change in a signature file, and then, in theory, you won't have to deal with the spaces here.

Reply:
Yes, correct. Not 100% safe, but it's where we want to end up for things like this, and we should try enabling it in some simpler cases. Here it only need to process whitespace between two tokens, so I'd try it. 🙂